Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -435,10 +436,11 @@ function xASL_wrp_Quantify(x, PWI_Path, pathOutputCBF, M0Path, SliceGradientPath
     fprintf('%s\n', 'Performing single PLD quantification');
     [~, CBF] = xASL_quant_SinglePLD(PWI, M0_im, SliceGradient, x, x.Q.bUseBasilQuantification); % also runs BASIL, but only in native space!
 	ATT = [];
+	TExch = [];
```

Comment:
We should come up with a default for 1) missing data and 2) processing going wrong, across the pipeline. BIDS has NaN or n/a for missing data/not available. So we could use 1) missing data == NaN or n/a 2) processing going wrong == []

Reply:
I've changed that to `NaN` already in 031559f8